README.md

```diff
@@ -146,6 +146,7 @@
 - [Silius - Rust Implementation](https://github.com/Vid201/silius/) - [blog](https://hackmd.io/@Vid201/aa-bundler-rust)
 - [TypeScript Implementation](https://github.com/eth-infinitism/bundler) by Infinitism
 - [Skandha - Typescript Implementation](https://github.com/etherspot/skandha) by Etherspot
+- [Transeptor - Typescript Implementation](https://github.com/transeptorlabs/transeptor-bundler) by Transeptor Labs
```

Comment:
great to see newcomers...
did you test it against the spec test? https://github.com/eth-infinitism/bundler-spec-tests/

Reply:
I have tested the bundler against the compatibility spec tests locally, and Transeptor is **not 100% compliant**. Below are the Transeptor `v0.1.0-alpha` results running with an up-and-running Ethereum node(geth) and bundler.

# Result running bundler in safe mode
Uses geth debug_traceCall method to enforce the full spec storage access rules and opcode banning.

<img width="1440" alt="safe-mode" src="https://github.com/4337Mafia/awesome-account-abstraction/assets/34751375/271048eb-a11a-475b-a26e-05a6d6e9753c">

# Results running bundler in unsafe mode
Standard call to entry Point Contract simulateValidation(). No storage access rules and opcode banning.

<img width="1440" alt="unsafe-mode" src="https://github.com/4337Mafia/awesome-account-abstraction/assets/34751375/73633483-261d-433d-b7ba-ec05fadc127f">

Reply:
Hey!,

If all is good then I'll proceed with a merge. Let me know, thanks!

Reply:
Hey, I just released `v0.2.0-alpha` for the Transeptor bundler. Still not 100% percent compliant but now passing all spec tests in safe mode *except for storage banning*. Test result details in release notes: https://github.com/transeptorlabs/transeptor-bundler/releases/tag/v0.2.0-alpha

Reply:
@drortirosh - when you have a sec to review! if good, we can merge